Epsilon.Host.Frontend/src/api.ts

```diff
@@ -106,16 +106,22 @@ export interface FullRequestParams extends Omit<RequestInit, "body"> {
     cancelToken?: CancelToken
```

Comment:
Revert these formatting changes, this file is generated.

Reply:
This file got added when I merged with development.

Reply:
formatting is prettier

Reply:
changed